DirectEmployers/MyJobs
# [PD-2980] nuo add text to email bucket

## What's this PR do?
This PR adds text onto the end of the tips section inside of inbox management for NUO

## Screenshots (if appropriate)
n/a

## Where should the reviewer start?
Start inside of NUO in the inbox management section

## How should this be manually tested?
Pull down the repository. When you go into the NUO application, click on Inbox Management on the right hand facet side. Inside you will see on the right hand side there is the original Tips along with text that's added to the end of it that should say "User sending email to these addresses will receive a confirmation message upon successfully submitting a NUP record."

## Any background context you want to provide?
Jason Sole said to add the text onto the end of the tips that were already there so it is concatenated inside of the pagetips in the navigation reducer.
